tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot

```diff
@@ -126,6 +126,49 @@ Add meta guidance to second Subject
     ...  meta guidance content
     user clicks button  Save guidance
 
+Navigate to 'Footnotes' page
+    [Tags]  HappyPath
+    user clicks link  Footnotes
+    user waits until h2 is visible  Footnotes
+
+Add footnote to Dates subject
+    [Tags]  HappyPath
+    user clicks link  Create footnote
+    user waits until h2 is visible  Create footnote
+    user clicks footnote checkbox  Select all indicators and filters for this subject
+    user enters text into element  id:footnoteForm-content  Footnote 1 dates subject
+    user clicks button  Save footnote
+    user waits until h2 is visible  Footnotes
+
+Add second footnote to Dates subject
+    user clicks link  Create footnote
+    user waits until h2 is visible  Create footnote
+    user opens details dropdown  Indicators
+    user clicks footnote checkbox  Admission Numbers
+    user enters text into element  id:footnoteForm-content  Footnote 2 dates subject
+    user clicks button  Save footnote
+    user waits until h2 is visible  Footnotes
+
+Add footnote to seven filters subject
+    [Tags]  HappyPath
+    user clicks link  Create footnote
+    user waits until h2 is visible  Create footnote
+    user clicks element  //*[@data-testid="footnote-subject Seven filters"]//input[@type="checkbox"]
```

Comment:
Should use the `${SUBJECT_NAME}` variable here instead of `Seven filters`.

I would replace all instances of `Seven filters` or `seven filters` with `${SUBJECT_NAME}`. You can also do this in test case names, so `Add footnote to seven filters subject` can become `Add footnote to ${SUBJECT_NAME} subject`.

You could do the same for all instances of `D/dates subject` with `${SECOND_SUBJECT}`.